### ISSUE TITLE
screen-readers don't read whether tests pass or fail

### ISSUE THREAD

**Issue by miki123211** [freeCodeCamp/freeCodeCamp#13013]
#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js
 
 
 
```
#### Screenshot

When we're editing a challenge, we have a list of tests read above the editor, but it's impossible to check if they pass or fail. I suggest adding a textual describtion i.e. 
passed: Make sure each of your a elements has a closing tag. 
or
failed: Make sure each of your a elements has a closing tag.

**Comment by Greenheart** [comment]
@miki123211 Thanks for reporting this!

I don't know much about accessibility, but would something like an attribute on the test results icon work?
I think there are some attributes made to help screen readers

**Comment by miki123211** [comment]
It might, but we would have to check. I can't even detect that the icon 
is here, as far as my screen-reader (NVDA) is concerned, there's 
nothing. No unlabeled graphics or image without an alt describtion, just 
the name of the test.
I'm using nvda and firefox, newest versions

**Comment by JosephBywater** [comment]
Would this help? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

**Comment by xnt** [comment]
@miki123211 I'm removing the icon and the text from the columns, and adding a concatenation of the result and the text as an aria-label to the row. I don't have NVDA at the moment, but Mac's VoiceOver reads it fine. Is there any chance you could help me test it?

**Comment by jhomme** [comment]
Hi,
I'm proud of myself for having gotten the site set up on my computer, because I was thinking of attempting to contribute, but now I think I need some hand-holding. I was going to test to see if this and other accessibility issues are fixed, but I'm unsure how to do that. I have multiple questions about what to do next. Where is the best place to go to discuss them?
